pusher/.env.example

```diff
@@ -0,0 +1,5 @@
+# could be quicknode, helius, triton, self-hosted etc
+RPC_URL=https://api.mainnet-beta.solana.com
```

Comment:
When I'm running the "test", I get

`Error: expected environment variable `ANCHOR_WALLET` is not set.`

or sth.

Where should I put these values? Any env?

Reply:
what is the exact command? I cannot find a single reference of `ANCHOR_WALLET` anywhere, it is not required, generally

Reply:
`yarn test` in the main directory

Reply:
`anchor test` should be used over `yarn test`, it injects a test wallet and starts a local validator node for the purpose of testing